lib/Doctrine/DBAL/Driver/V3CompatStatement.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace Doctrine\DBAL\Driver;
+
+use Doctrine\DBAL\Abstraction\V3Result;
+use Doctrine\DBAL\Exception;
+use IteratorAggregate;
+use PDO;
+use Traversable;
+
+class V3CompatStatement implements IteratorAggregate, ResultStatement, V3Result
```

Comment:
Please move to a separate `ForwardCompatibility` namespace.

Reply:
Done